### PR TITLE
Ui design

### PR DESCRIPTION
**Pull request by patelravi**

### CODE REVIEW

File: front/.babelrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "presets": [
-    "vue-app"
+    "vue-app-bar"
```
Comment (aldarund) [inline front/.babelrc]: what its for? there no such preset as vue-app-bar

File: front/pages/ideas/_userId/_ideaId.vue [hunk 2]
```diff
@@ -407,33 +492,138 @@ export default {
...
-      'illustration',
-      'graphics',
-      'ui',
-      'adobe',
-      'interface'
-    ]
+    let isLiked = await app.$amplifyApi.graphql(
+      graphqlOperation(getIsIdeaLikedByMe, { ideaId: route.params.ideaId })
+    )
+    isLiked = isLiked.data.getIsIdeaLikedByMe.isLiked
```
Comment (aldarund) [inline front/pages/ideas/_userId/_ideaId.vue]: reassigning like this isnt good. You should use desctuctring if u want get that https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

File: front/.babelrc (hunk 1 above)
Comment (patelravi) [reply front/.babelrc]: Resolved.

File: front/pages/ideas/create.vue [hunk 3]
```diff
@@ -50,7 +64,7 @@
 
         <!-- Submit -->
         <div class="submitBtn">
-          <v-btn :loading="creatingIdea" @click="onCreateIdea">Submit</v-btn>
+          <v-btn :loading="creatingIdea" @click="onCreateIdea()">Submit</v-btn>
```
[the remark this answers is not in the document]
Comment (patelravi) [reply front/pages/ideas/create.vue]: Resolved.

**Pull request merged**